astro/src/content/docs/lifecycle/migrate-users/bulk/stytch.mdx

```diff
@@ -0,0 +1,327 @@
+
+## Get All User Details And Combine With Hashes
+
+Stytch emails you only the user hash and ID. You need to use the Stytch API to retrieve all user details from Stytch, then combine those with the hash from the email, and save the user to FusionAuth.
+
+Open your hash file from Stytch in a text editor and remove all the header lines so only one user per row remains. Save this file to `stytch/js/hash.csv`. The file should look like `exampleData/4_hashFilePreparation/hash.csv` now.
+
+<RemoteCode
+    url={frontmatter.importCodeRoot + "/stytch/exampleData/4_hashFilePreparation/hash.csv"}
+    lang="csv"
+/>
+
+Check the fourth column to ensure that only scrypt users are included. The first four columns are most important: `id`, `hash`, `salt`, and `hash_method`.
+
+The `3_getUserDetails.mjs` script loops through each user in the `hash.csv` file, uses the API to get the user details from Stytch, and saves them to `js/users.json`.
+
+- Open `3_getUserDetails.mjs` and set your project Id and secret.
+- Run the code below in a terminal in the `fusionauth-import-scripts/stytch/js` directory.
+  ```bash
+  npm update
+  node 3_getUserDetails.mjs
+  ```
+
+Open `js/users.json`. It should look like the sample in `exampleData/5_userDetailAndHashPreparation/users.json`.
+
+<RemoteCode
+    url={frontmatter.importCodeRoot + "/stytch/exampleData/5_userDetailAndHashPreparation/user.json"}
+    lang="json"
+/>
+
+<Aside type="note">
+In Stytch, a user can have multiple emails and phone numbers. FusionAuth allows for only one. You will store the extras in the user data JSON field when migrating the user, along with any other Stytch fields FusionAuth does not support.
+</Aside>
+
+The last lines of the object show the hash and salt that the script added from the hash file to the user details from the API.
+
+## Install The Scrypt Password Hash Plugin For FusionAuth
+
+The scrypt hashing algorithm is not [natively supported by FusionAuth](/docs/reference/password-hashes). However, FusionAuth allows [custom plugins](/docs/extend/code/password-hashes/custom-password-hashing). There is a scrypt plugin accompanying this article [in this GitHub repository](https://github.com/FusionAuth/fusionauth-contrib/tree/master/Password%20Hashing%20Plugins).
+
+<Aside type="note">
+The linked repository has multiple plugin projects. You need to build and install only the hashing project.
+</Aside>
+
+- Download and unzip the repository, or use Git in a terminal with the code below.
+  ```bash
+  git clone https://github.com/fusionauth/fusionauth-contrib.git
+  ```
+
+Open the file `fusionauth-contrib/Password Hashing Plugins/src/main/java/com/mycompany/fusionauth/plugins/ExampleStytchScryptPasswordEncryptor.java`. The content of the file is shown below.
+
+<RemoteCode
```

Comment:
showing 404 code found error. Is this something that needs to be merged?

Reply:
So is that PR ready for review as well?

Reply:
@mark-robustelli - yes that one is also ready for review. We usually just mention any other scripts/repos etc as a dependency in the top-level comment in the main PR to fusionauth-site as we did here, but let me know if there's a better way to do this